contracts/Hello.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.5.0;
+
+contract Hello {
+    string public message;
+
+    constructor() public {
+        message = "Hello world!";
+    }
+}
```

Comment:
Will we remove this contract again once deployment works reliably?

Reply:
We could. Or we could keep it and test that a call to `message` returns the hello world string. It's true that we don't need this contract to demonstrate the functionality. It's just a template.

Reply:
I added such test in https://github.com/poanetwork/posdao-test-setup/pull/15/commits/13aa4ea6b6681a9a2e24011d348e39c97b559d86.